src/docs/sphinx/dev_guide/updating_tpls.rst

```diff
@@ -101,68 +107,76 @@ Axom only uses it for our Windows TPL builds.
+#. Add Axom's package repository to Spack, so our packages take precedence.
+#. Clean temporary information from previous runs that may bleed into a new run.
 #. Optionally create a package source mirror.
 #. Install packages via the selected package manager.
 
-* ``.uberenv_config.json``: This file describes project specific configurations,
-  such as, where to download the package manager, what git commit to use, and
-  the top level package to install.
-* `Uberenv's GitHub repo <https://github.com/LLNL/uberenv>`_
-* `Uberenv's documentation <https://uberenv.readthedocs.io/en/latest/>`_
+* The information provided to Uberenv to start this workflow is defined in one file in
+  the top-level Axom source directory:
+
+   * ``.uberenv_config.json``: This file describes project specific configurations,
+     such as where to download the package manager, what git commit to use, and
+     the top level package to install.
+
+* More detailed information about Uberenv can be found in the 
+  `Uberenv GitHub repo <https://github.com/LLNL/uberenv>`_
+  and in the `Uberenv documentation <https://uberenv.readthedocs.io/en/latest/>`_
 
 .. note::
-   Uberenv's warnings and errors are easy to find by searching the output for ``[ERROR:``
-   or ``[Warning:``.  Uberenv will stop at the first error.
+   Uberenv warnings and errors are easy to find by searching the output for ``[ERROR:``
+   or ``[Warning:``.  Unlike Spack, Uberenv will stop at the first error it encounters.
 
 Level 3: Build Scripts
 ----------------------
 
 The file ``axom/scripts/spack/specs.json`` contains a list of all specs
-required per platform or machine name. These specs automatically handle
+that we share for Axom development and GitLab CI testing on the LC platforms
+we use for development and testing. The specs automatically handle
 platform differences and contain the full list of compilers and package specs
 required to build.
 
 The directory ``axom/scripts/llnl_scripts`` contains three "build" scripts that
-are designed to handle building suites of TPLs via Uberenv and Spack.
+are designed to build suites of TPLs via Uberenv and Spack.
 
-* ``build_tpls.py``: This script starts by building all TPLs listed in the file
-  ``specs.json``. It will generate host-config files and copy them to the base
-  of the Axom repository. After building all of the TPLs, it will test Axom
-  against those built TPLs as well as test the installed ``using-with-cmake``
+* ``build_tpls.py``: First, this script builds a set of TPLs for each of the specs
+  listed in the ``specs.json`` file for the platform on which it is run. For each TPL set,
+  it will generate a host-config file and copy it to the top-level directory of the local
+  copy of the Axom repository in which the script is run. After building all of TPL sets
+  for a platform, it will test Axom against those TPL installs as well as the ``using-with-cmake``
   example for correctness. This script stops at the first failed TPL build but
```

Comment:
```suggestion
  copy of the Axom repository in which the script is run. After building all TPL sets
  for a platform, it will attempt to build Axom and the ``using-with-cmake`` example against 
  each set of TPLs. This script stops at the first failed TPL build but
```

Reply:
Recommended change pushed.